### DISCUSSION TITLE
Which translator is integrated in the module?

### DISCUSSION THREAD

**Discussion by Herzfrequenz21** [Purfview/whisper-standalone-win#244] [Q&A] [answered]
On your advice, I'm opening a new thread.

Can you tell me which translator is integrated in the module?

**Comment by Purfview** [comment, marked as answer]
There is no external translator integrated. Same whisper model does transcription and translation.

**Reply by Herzfrequenz21** [reply]
But Whisper has that model from somewhere, right?
Or did you write the algorithm yourself?

For example, if you take 10 online translators, 7-8 of them will have identical translations, because they use the same model (engine, algorithm - I don't know how to call it right).

Ok, is there an online translator for your whisper? To see how it translates.

**Reply by Purfview** [reply]
>But Whisper has that model from somewhere, right?

Models are made by OpenAI, same who made Whisper.

>Ok, is there an online translator for your whisper? To see how it translates.

There is no separate model just for translation, it's same model for transcription and translation.
